docs/wsrep-system-index.md

```diff
@@ -1315,6 +1315,8 @@ and manually restoring the same data to all nodes.
 This option should not be used permanently
 because it could lead to data inconsistency across the nodes.
 
+* `ist_only` : **Introduced in Percona XtraBackup 8.0.33-25.** This value allows only Incremental State Transfer (IST). If a node cannot sync with the cluster with IST, abort that node's start. This action leaves the data directory unchanged. This value prevents starting a node, after a manual backup restoration, that does not have a `grastate.dat` file. This missing file could initiate a full-state transfer (SST) which can be a more time and resource-intensive operation.
```

Comment:
Introduced in Percona XtraBackup -> should it be Percona XtraDB Cluster?

Reply:
Fixed.